.github/actions/changed-packages/action.yml

```diff
@@ -0,0 +1,38 @@
+# Example usage in `./workflows/reusable-e2e.yml`
+name: Check Changed Packages
```

Comment:
This action may come in handy for the future automated changesets checker workflow.

Reply:
Really appreciate this action, but wondering if it should be in a follow up where we address all platforms?

Reply:
Not sure I understand, this action is needed here to make sure the react-native tests don't run if non related packages are modified.

Reply:
Sentiment was that this would be great to have for all supported platforms (for example Vue), so maybe should be done separately. Although i get it if you want to only have it for RN immediately since the test take so much time to run, and then expand it for other platforms as a separate task

Reply:
Yes that was the idea, we would want to have it for RN immediately, we will add docs, other platforms as follow-up